InMobiAdsDemo/InMobiAdsDemo/Controllers/IMNativeViewController.m

```diff
@@ -0,0 +1,243 @@
+//
+//  IMStrandsViewController.m
+//  Ads Demo
+//
+//  Copyright © 2016 Inmobi. All rights reserved.
+//
+
+#import "IMNativeViewController.h"
+#import <InMobiSDK/InMobiSDK.h>
+#import "IMSimpleTableCell.h"
+#import "IMAppIds.h"
+
+#define kIMAdInsertionPosition 5
+@interface IMNativeViewController() <IMNativeDelegate,UITableViewDelegate,UITableViewDataSource>
+@property (nonatomic,strong) IMNative *native;
+@property (nonatomic) BOOL nativeStrandsLoaded,nativeStrandsRendered,nativeStrandsInserted;
+@property (nonatomic,strong) UITableView *tableView;
+@property (nonatomic,strong) NSMutableArray *tableData;
+@end
+@implementation IMNativeViewController
+- (void)viewDidLoad {
+    self.tableData = [[NSMutableArray alloc] init];
+    [self loadInitialData];
+    [self loadInitialData];
+    if(self.isStoryBoard){
```

Comment:
can be written as 
`long long placementId = self.isStoryBoard ? INMOBI_NATIVE_STORYBOARD : INMOBI_NATIVE_MOVIEBOARD;
 self.native = [[IMNative alloc] initWithPlacementId:placementId];`

Reply:
done